low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -32,18 +39,133 @@ class ServerMap(ResourceMap):
 
     def __init__(self):
         """Initialize server"""
-        pass
+        super().__init__()
+        self.sysfs = ToolFactory().get_instance('sysfs')
```

Comment:
Can we put this sysfs init in a function in common utils, like mon utils and call from there